core/build.gradle

```diff
@@ -21,6 +21,7 @@ managedDependencies {
         compile "io.netty:$it"
     }
     compile 'io.netty:netty-transport-native-epoll:linux-x86_64'
+    compile 'io.netty:netty-transport-native-kqueue:osx-x86_64'
```

Comment:
I don't feel super-strongly either. FWIW, my idea was something along the lines of this in the gradle file

```
if (os == mac)
  compile 'kqueue'
```

This would enable kqueue during development on macs (think it'd work for intellij too) though requiring an extra step if the user did actually want to deploy to a macos server.